developer-docs-site/docs/concepts/coin-and-token/mint-minting-tool.md

```diff
@@ -0,0 +1,91 @@
+# NFT Minting Tool CLI
+
+> The tool is tested on Mac. It should work on Linux and WSL too.
+
+# Setup
```

Comment:
If this is adding minting CLI for everyone to use, we should test it on windows as well.